pages/asctb-azimuth-data-comparison.html

```diff
@@ -208,12 +222,41 @@ <h3 class="mt-5">Comparison of CxG in ASCT+B vs. Azimuth</h3>
                                             <td class="text-right">0</td>
                                             <td class="text-right">0</td>
                                             <td class="text-right">0</td>
+                                            <td class="text-right">N/A</td>
                                             <td class="text-right">1</td>
                                             <td class="text-right">35,289</td>
                                             <td class="text-right">128</td>
                                             <td class="text-right">0</td>
                                             <td class="text-right">0</td>
                                         </tr>
+                                        <tr>
+                                            <td class="align-top"><a href="https://hubmapconsortium.github.io/asctb-azimuth-data-comparison/spleen.csv" title="Spleen csv file">Spleen</a></td>
+                                            <td class="text-right">49</td>
+                                            <td class="text-right">N/A</td>
+                                            <td class="text-right">66</td>
+                                            <td class="text-right">N/A</td>
+                                            <td class="text-right">N/A</td>
+                                            <td class="text-right">4</td>
+                                            <td class="text-right">9</td>
+                                            <td class="text-right">5,104</td>
+                                            <td class="text-right">N/A</td>
+                                            <td class="text-right">N/A</td>
+                                            <td class="text-right">N/A</td>
+                                        </tr>
+                                        <tr>
+                                            <td class="align-top"><a href="https://github.com/hubmapconsortium/asctb-azimuth-data-comparison/blob/gh-pages/Human%20Lung%20v2.celltype_stats.csv" title="Lung v2 csv file">Lung v2</a></td>
```

Comment:
Use the other url pattern here:
<https://hubmapconsortium.github.io/asctb-azimuth-data-comparison/Human%20Lung%20v2.celltype_stats.csv>

Also, it'd be better to rename to remove uppercase letters and replace space with dash.

Reply:
@Sourolio10 Use the other url pattern here:
https://hubmapconsortium.github.io/asctb-azimuth-data-comparison/Human%20Lung%20v2.celltype_stats.csv

Also, it'd be better to rename to remove uppercase letters and replace space with dash.